app/controllers/submitted_content_controller.rb

```diff
@@ -57,6 +58,23 @@ def submit_hyperlink
       end
       ExpertizaLogger.info LoggerMessage.new(controller_name, @participant.name, 'The link has been successfully submitted.', request)
       undo_link("The link has been successfully submitted.")
+
+      # E1834 Fall 18
+      # Send email to reviewers to review new submission, if review_round is valid and not last.
+      if participant.is_round_valid_for_mail?
+        #reviewers = []
+        participant.reviewers.each do |reviewer|
+          #reviewers << User.find(reviewer.user_id).email
+          review_mappings = ResponseMap.where(reviewer_id: reviewer.id)
+          review_mappings.each do |map|
+            reviewed_object_id = ''
+            if map.reviewee_id == @participant.id || map.reviewee_id == @participant.team.id
```

Comment:
On further inspection, we found that the id passed in the URL suffix was incorrect. We referred to reviewed_object_id instead of lastest_response.id. 
We have corrected it now and the new code changes make it much better. 
Commit id: e09761b7d8746f26485f264c7fd7936b8d956326